<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Check</class>
 <widget class="QDialog" name="Check">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 243, 244)</string>
  </property>
  <widget class="QPushButton" name="odgovori">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>260</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Odgovori</string>
   </property>
  </widget>
  <widget class="QLabel" name="pitanje">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>301</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="odg1">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="odg2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="odg3">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="odg4">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="odg5">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
